sdb/commands/linux/dmesg.py

```diff
@@ -23,18 +23,17 @@
 
 
 class DMesg(sdb.Locator, sdb.PrettyPrinter):
-    # pylint: disable=too-few-public-methods
 
     names = ["dmesg"]
 
     input_type = "struct printk_log *"
     output_type = "struct printk_log *"
 
     def no_input(self) -> Iterable[drgn.Object]:
-        log_idx = sdb.prog["log_first_idx"]
-        log_seq = sdb.prog["clear_seq"]
-        log_end = sdb.prog["log_next_seq"]
-        log_buf = sdb.prog["log_buf"]
+        log_idx = sdb.get_object("log_first_idx")
+        log_seq = sdb.get_object("clear_seq")
+        log_end = sdb.get_object("log_next_seq")
+        log_buf = sdb.get_object("log_buf")
```

Comment:
how about renaming `get_object` to `get_symbol`?

Reply:
Unfortunately, if you look at `target.py` you will see the following:
```
def get_object(obj_name: str) -> drgn.Object:
    global prog
    return prog[obj_name]


def get_symbol(sym: Union[str, int]) -> drgn.Symbol:
    global prog
    return prog.symbol(sym)
```
One returns a `drgn.Object` the other one a `drgn.Symbol` and their argument is slightly different (e.g. you can get a symbol by its address but not an object) - I decided to have them modeled after what they return from `drgn` as so far `target.py` is mostly wrappers around it. If you think we could do a better job and combine the above too or just rename them to something more precise or intuitive I'm open to ideas!